### PR TITLE
Aws iam console access2

### LINKED ISSUE #620: Add/Enable AWS Console Access View to the EKS cluster for the observability workshop
### What would you like to be added?

Withing the Observability module, the module jumps directly to the AWS EKS Console to view the different resources of the cluster. However, to be able to see the information you need to grant permissions to the IAM Role the AWS Console.

image

### Why is this needed?

Withing the EKS cluster deployment grant the Console access or add a command to prepare the environment with he right access for the console.

Create the RBAC role and rolebinding:
```
curl -o eks-console-full-access.yaml https://amazon-eks.s3.us-west-2.amazonaws.com/docs/eks-console-full-access.yaml
```

Grant permissions to the IAM role to view the information with the specified group previously created:
```
eksctl create iamidentitymapping --cluster [eks-cluste-name] --region=[aws-region] \
    --arn arn:aws:iam::[accountId]:role/[role-name] --username aws-console-role --group eks-console-dashboard-full-access-group \
    --no-duplicate-arns
```

### PR DESCRIPTION
**Pull request by arcegacardenas**
#### Which issue(s) this PR fixes:
It will grant access to the EKS through the AWS console when the user launched the cluster through Cloud9.

Fixes #620

#### Quality checks

- [x] My content adheres to the style guidelines
- [x] I ran `make test` or `make e2e-test` and it was successful

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: manifests/modules/observability/base/.workshop/terraform/addon.tf [hunk 1]
```diff
@@ -0,0 +1,118 @@
+resource "kubernetes_cluster_role" "eks-console-dashboard-full-access-clusterrole" {
```
Comment (niallthomson) [inline manifests/modules/observability/base/.workshop/terraform/addon.tf:1]: Is this file still needed with the logic in `use-cluster`? Will it not do the same thing? Or is there something else?

File: hack/run-tests.sh [hunk 2]
```diff
@@ -54,5 +54,5 @@ echo "Running test suite..."
 $CONTAINER_CLI run $background_args \
   -v $SCRIPT_DIR/../website/docs:/content \
   -v $SCRIPT_DIR/../manifests:/manifests \
-  -e 'EKS_CLUSTER_NAME' -e 'AWS_REGION' \
+  -e 'EKS_CLUSTER_NAME' -e 'AWS_REGION' -e 'C9_USER' \
```
Comment (niallthomson) [inline hack/run-tests.sh:57]: Is this designed to cater for the code being developed in Cloud9? If I'm developing on my local laptop will this skip triggering the Cloud9 logic in the rest of the change?

File: manifests/modules/observability/base/.workshop/terraform/addon.tf (hunk 1 above)
Comment (arcegacardenas) [reply manifests/modules/observability/base/.workshop/terraform/addon.tf:1]: Forgot to delete those files. I isn't needed.

File: hack/run-tests.sh (hunk 2 above)
Comment (arcegacardenas) [reply hack/run-tests.sh:57]: I needed it to emulate the C9 deployment in my local computer.  But might be unnecessary, I'll do a test and remove it if it is not necessary.

File: manifests/modules/observability/base/.workshop/terraform/addon.tf (hunk 1 above)
Comment (arcegacardenas) [reply manifests/modules/observability/base/.workshop/terraform/addon.tf]: Unnecessary files removed.

File: hack/run-tests.sh (hunk 2 above)
Comment (arcegacardenas) [reply hack/run-tests.sh]: Variable removed
